app/helpers/application_helper.rb

```diff
@@ -13,6 +13,21 @@ def markdown(source)
     options = { autolink: true, lax_spacing: true }
     markdown = Redcarpet::Markdown.new(render, options)
     markdown.render(source).html_safe
+
+    # Fix common markdown errors:
+    # - using bullets rather than *
+    # - not putting a space between * and word
+    source = source.gsub(/•\s?/, "* ").gsub(/^\*(?![\s\*])/, "* ")
+
+    # Convert quotes to smart quotes
+    source_with_smart_quotes = smart_quotes(source)
+    markdown.render(source_with_smart_quotes).html_safe
+  end
```

Comment:
I'd purposefully omitted this from Publish preview, as I'd prefer providers to put in the correct markdown, and then fix it if it's broken when checking on preview.

Reply:
i see your point but at the same time don't users want to see the actual preview? I guess it's a double edged sword. I think i will leave smart quotes in and remove bullet point one

Reply:
line 20 has now gone